guacapy/client.py

```diff
@@ -77,7 +83,7 @@ def __authenticate(self):
             allow_redirects=True,
         )
         r.raise_for_status()
-        return r.json()
```

Comment:
Why the change?

Reply:
This done to get cookies data from response. In earlier case,  I was not able to find cookies content in r.json().